### DISCUSSION TITLE
Expected release date.

### DISCUSSION THREAD

**Discussion by wertone-gh** [Try/OpenGothic#245] [General]
Hello, I wonder when do you expect release build to be ready?
I really wish to play G2 on x64 native ;3
Current builds are still marked as prerelease?
How much work is still needed to be done to deliver release build?
Regards and thank you for your effort.

**Comment by Try** [comment]
Hi, @wertone-gh !

That quite a big question, I'll do proper list of tickets that we need before release. 
Until then, short answer:
Game is pretty much playable, missing only some UI stuff and weather effects.

**Comment by Try** [comment]
I've created a new GitHub-milestone 1.0:
https://github.com/Try/OpenGothic/milestone/1
That will give fair perspective on what is missing. Probably I'll add more tickets to that in time, yet that all what I can remember so far.

**Reply by thedrakeraider** [reply]
Will it support Gothic 1 at 1.0 as well? I'm quite excited for this. I downloaded Chronicles of Myrtana in the meantime, but my heart's in the main series.

**Reply by Try** [reply]
No, not in 1.0. G1 will require extra api entries for scripting, extra combat mechanics and magic-barrier support.
Also probably CM support is not going to meet 1.0 date - since it's non-confomant mod

**Reply by thedrakeraider** [reply]
I just realized I never actually ask _when_ Gothic 1 support is expected? I'm sorry that you probably get this question alot.

**Reply by Try** [reply]
Hi, @thedrakeraider !

There is no strong deadline to that. Also I'm myself is more gothic2 person - so yeah, not soon

**Reply by thedrakeraider** [reply]
I'm definitely a Gothic 2 person myself, so I understand, I'm just excited for a chance to play the whole series through in native for linux!

Thanks for the response, I really appreciate it.

**Reply by wertone-gh** [reply]
Any chance for stable (G2) release this year?

**Reply by Try** [reply]
If you don't count full marving support - there is 2 things remain: weather and ladder. So it's quite realistic to finish this year.
Apart from that working in testing current build, with release plans to the end of June.
